local.yml

```diff
@@ -89,3 +92,29 @@ services:
     ports:
       - "5555:5555"
     command: /start-flower
+
+  elasticsearch:
+    container_name: elasticsearch
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.8.1
+    environment:
+      - node.name=es
+      - cluster.name=es-docker-cluster
+      - discovery.type=single-node
+      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+    volumes:
+      - es-data:/usr/share/elasticsearch/data
+    # command to limit logging messages of starting service
+    command: ["elasticsearch", "-Elogger.level=WARN"]
+    expose:
+      - 9200
+    networks:
+      - elastic
+
+networks:
+  elastic:
+    driver: bridge
```

Comment:
why bridge?

Reply:
I found [this](https://www.elastic.co/blog/docker-networkingl)
so there was 2 options (bridge - overlay)
since we are not using swarm mode or multi-host , overlay won't get advantage.